lib/tasks/publishing_api.rake

```diff
@@ -25,6 +25,25 @@ namespace :publishing_api do
     end
   end
 
+  desc "Take over ownership of a route (used for migrating finder from Whitehall)"
+  task :claim_route do
+    route_to_claim = ENV["FINDER_BASE_PATH"]
+
+    unless route_to_claim
+      raise "Please supply a valid base path to claim, e.g. FINDER_BASE_PATH=/government/case-studies"
+    end
+
+    puts "Claiming route #{route_to_claim}..."
+    Services.publishing_api.put_path(
+      route_to_claim,
+      {
+        publishing_app: "search-api",
+        override_existing: true,
+      },
+    )
+    puts "Route claimed."
+  end
```

Comment:
It'd be good to have a test of this for code coverage and to hopefully catch if Publishing API had any API changes